routes/api/v1/favorites.js

```diff
@@ -0,0 +1,37 @@
+var express = require("express");
+var fetch = require("node-fetch");
+var router = express.Router();
+var user = require('../../../models').User
+var favorite = require('../../../models').Favorite
+
+router.post("/", function(req, res, next) {
+  res.setHeader("Content-Type", "application/json");
+  let city = req.body.location
+
+  if (!req.body.apiKey) {
```

Comment:
What if no location is sent in the body either? Might be good to add this extra argument to make sure all of your sad paths are covered

Reply:
Thanks for the feedback Tay!  I've updated this request so that my error messages are more clear and I am checking for a missing location as well as api key.

Reply:
Thanks for the feedback Tay!  I've updated this request so that my error messages are more clear and I am checking for a missing location as well as api key.